### DISCUSSION TITLE
lab-snps-flow problem

### DISCUSSION THREAD

**Discussion by sungsungwu** [bol-edu/HLS-SOC-Discussions#208] [Q&A]
請問有人在執行此lab的lab_1_step3_power plan時有遇到以下問題嗎?
![problem](https://github.com/bol-edu/HLS-SOC-Discussions/assets/113113868/313f64e8-d3ff-4eef-80d5-1ab848aaaae2)
上面寫的第377行我有去查有沒有錯誤資料，但似乎沒有寫到為何會報錯
![p](https://github.com/bol-edu/HLS-SOC-Discussions/assets/113113868/ecb0568e-1d2f-4336-beca-973ef2dbe05b)
![p2](https://github.com/bol-edu/HLS-SOC-Discussions/assets/113113868/35af3482-f211-4b47-80bd-b7fcfa70fa56)
step2最後的結果如下似乎也沒有報甚麼錯誤。
![s2](https://github.com/bol-edu/HLS-SOC-Discussions/assets/113113868/a5c5c5ab-3d5b-46f1-a7fd-2530dde580d8)

我們也有從網路上找相關資料也有使用64bit來執行，清大電機工作站的centOS機台也有切換過了，仍然沒辦法解決，請問也有其他人遇到此問題知道該如何解決嗎? 感謝各位

**Comment by coo122567** [comment]
同學你好:
我也遇到此問題，我的解決方法是更換Tool的version.
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/118537645/f62cb1db-d879-4e4c-8a5d-246b2b06da5f)
你可以試試看!

**Reply by sungsungwu** [reply]
同學你好:
![image](https://github.com/bol-edu/HLS-SOC-Discussions/assets/113113868/117bcb00-807c-49f9-bd96-fe311cc34adf)
按照同學的方法做成功了，感謝!
